go/metadata/capabilities/capabilities.go

```diff
@@ -221,14 +206,43 @@ func normalize(in map[string]interface{}) (map[string]interface{}, error) {
 	}
 
 	for k, v := range in {
-		if k != "chromeOptions" && k != "goog:chromeOptions" && k != "proxy" {
+		if k != "chromeOptions" && k != "goog:chromeOptions" && k != "proxy" && k != "loggingPrefs" && k != "goog:loggingPrefs" {
 			out[k] = v
 		}
 	}
 
 	return out, nil
 }
 
+// normalizeParam copies and merges paramName from "in" to "out" with name "goog:"+paramName.
+func normalizeParam(in map[string]interface{}, out map[string]interface{}, paramName string) (map[string]interface{}, error) {
```

Comment:
This is not a good name. Maybe duplicateLegacyGoogCapability?

Reply:
Done.